mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -105,9 +105,17 @@
     <!-- Text for the negative action button to dismiss the Close Tabs Banner. -->
     <string name="tab_tray_close_tabs_banner_negative_button_text">Dismiss</string>
     <!-- Text for the banner message to tell users about our inactive tabs feature. -->
-    <string name="tab_tray_inactive_onboarding_message" tools:ignore="UnusedResources">Tabs you haven’t viewed for two weeks get moved here.</string>
+    <string name="tab_tray_inactive_onboarding_message">Tabs you haven’t viewed for two weeks get moved here.</string>
     <!-- Text for the action link to go to Settings for inactive tabs. -->
-    <string name="tab_tray_inactive_onboarding_button_text" tools:ignore="UnusedResources">Turn off in settings</string>
+    <string name="tab_tray_inactive_onboarding_button_text">Turn off in settings</string>
+    <!-- Text for title for the auto-close dialog of the inactive tabs. -->
+    <string name="tab_tray_inactive_auto_close_title">Auto-close after one month?</string>
+    <!-- Text for the body for the auto-close dialog of the inactive tabs. -->
+    <string name="tab_tray_inactive_auto_close_body">Firefox can close tabs you haven’t viewed over the past month.</string>
```

Comment:
Seems like we're starting to hardcode "Firefox" a lot since the last strings PR, just wanted to call out that this make repacking/rebranding impossible.
Perhaps a good idea to replace with a placeholder instead, like we used to do (and update the occurrences of Firefox in existing strings, if needed)
CC @Amejia481 for input

Reply:
What about repacking?
I'll let Betsy chime in, this isn't urgent at this point

Reply:
Sorry, could you help me to clarify what it's repacking? 😄  I'm not familiar with the term.

Reply:
oh sorry, I meant for in the case that people recompile the code and make changes to it. Which in this case they would need to rename the product if they want to redistribute it, as per our policy :)

Reply:
That could be a good use case 👍🏽